### PR TITLE
Fix #729: Simplify and fix Vite Code Start

### LINKED ISSUE #729: CodeStart does not work out of the box
### Describe the bug

CodeStart running `mvn quarkus:dev` does not serve the app.

### Quinoa version

2.4.3

### Quarkus version

3.12.3

### Build / Runtime

Vite

### Package Manager

NPM

### Steps to reproduce the behavior

1. https://code.quarkus.io/
2. Pick only Quinoa and run `mvn quarkus:dev` it does not run the Vite site at `localhost:8080`
3. 
image

### Expected behavior

The app should work out of the box

### PR DESCRIPTION
**Pull request by melloware**
Fix #729: Simplify and fix Vite Code Start

image

### CODE REVIEW

File: integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java [hunk 1]
```diff
@@ -8,26 +8,26 @@
 
 import io.quarkus.devtools.testing.codestarts.QuarkusCodestartTest;
 
-public class QuinoaCodestartTest {
+public class QuinoaCodeStartTest {
```
Comment (ia3andy) [inline integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java:11]: ??? :)

**Comment by ia3andy** [comment]
What no more Matrix 🤣

File: integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java (hunk 1 above)
Comment (melloware) [reply integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java:11]: Sorry the case seemed weird to me that its CodeStart but the s was lowercase it looked weird to my eyes :)

**Comment by melloware** [comment]
HA yeah sorry about that but people were confused why the starter used `express.js` etc so this one has a single dependency `vite` and i think boils it down nicely.

File: integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java (hunk 1 above)
Comment (ia3andy) [reply integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java:11]: Ok we use Codestart as a single word (a codestart) everywhere, so better keep it that way..

**Comment by ia3andy** [comment]
@melloware I am not sure why it was depending on express, weird... but alright as you see fit.

File: integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java (hunk 1 above)
Comment (melloware) [reply integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java:11]: OK let me revert that change then.  I didn't realize that it just look funny to me!

**Review by ia3andy — APPROVED** [review]
Thanks!!

**Pull request closed without merging**